web-test-runner.config.js

```diff
@@ -0,0 +1,84 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+const { createSauceLabsLauncher } = require('@web/test-runner-saucelabs');
+const { legacyPlugin } = require('@web/dev-server-legacy');
+const { playwrightLauncher } = require('@web/test-runner-playwright');
+
+const baseConfig = {
+  files: '{core, dataviz, ui}/test/**/*.test.js',
+  groups: [
+    {
+      name: 'core',
+      files: 'core/test/**/*.test.js',
+    },
+    {
+      name: 'dataviz',
+      files: 'dataviz/test/**/*.test.js',
+    },
+    {
+      name: 'ui',
+      files: 'ui/test/**/*.test.js',
+    },
+  ],
+  browsers: [
+    playwrightLauncher({ product: 'chromium' }),
+    playwrightLauncher({ product: 'webkit' }),
+    playwrightLauncher({ product: 'firefox' }),
+  ],
+  coverage: true,
+  nodeResolve: true,
+  testFramework: {
+    config: {
+      ui: 'tdd',
+    },
+  },
+};
+
+const isSauceLabsRun = process.env.SAUCE_USERNAME && process.env.SAUCE_ACCESS_KEY;
+if (isSauceLabsRun) {
+  const sharedCapabilities = {
+    'sauce:options': {
+      name: 'Nuxeo Cold Storage',
```

Comment:
guess we need a more generic name here ;)

Reply:
Caught! Damn copy paste 😂